data/custom.xml

```diff
@@ -39,7 +39,7 @@
 
     <magic priority="50">
       <match value="PK\003\004" type="string" offset="0">
-        <match value="[Content_Types].xml" type="string" offset="30">
+        <match value="[Content_Types].xml" type="string" offset="30:65536">
```

Comment:
The composing parts of google-generated office documents seems to be order-free. As the file gets longer, the magic bytes come in later - meaning, we have to search longer bytes.

pptx naturally has larger components (even though test fixture is nothing more than a blank slide), so it has to be at least 30:28168 to find out the string. To make the numbers less cryptic, I used a common range for the three different format.

Looks like we have two choices here:
1. to align with mimemagic that searches the string upto ~5000 bytes. In this case we cannot add google-generated pptx fixture, as it naturally fails.

2. to take bigger range than mimemagic does, so that it can identify (at least) some office files.

I'm assuming the latter might make sense, but please feel free to correct me if I'm wrong.

In any case we might have to rely on other fallback strategies for larger google-generated office files, so probably the question would come down to "to what extent do we want to try hard to find out embedded magic string?" - let me know what you would think.